packages/blade/src/components/Alert/_decisions/decisions.md

```diff
@@ -0,0 +1,162 @@
+| variant       | `info`, `positive`, `notice`, `negative` | `info`      | Sets the color tone of entire alert. Icon is set automatically based on variant \*         |          |
+| isBorderless  | `boolean`                                | `false`     | Removes borders and border radii, useful for creating a full bleed layout                  |          |
+
+We'll also expose the following sub components on `Alert` which can be passed as children to `Alert`. The order of passing these as children doesn't matter:
+
+| Sub component           | Description                                                          | Required |
+| ----------------------- | -------------------------------------------------------------------- | -------- |
+| `Alert.Description`     | For setting body content                                             | ✅       |
+| `Alert.PrimaryAction`   | Renders a primary `Button`                                           |          |
+| `Alert.SecondaryAction` | Renders a button `Link`, requires `Alert.PrimaryAction` alongside \* |          |
+
+### Alert.Description
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content \*  | ✅       |
+
+### Alert.PrimaryAction
+
+`Alert.PrimaryAction` renders and accepts a subset of `Button` (primary) props:
+
+| Prop       | Type       | Default     | Description                            | Required |
+| ---------- | ---------- | ----------- | -------------------------------------- | -------- |
+| `children` | `string`   | `undefined` | Content, required here unlike `Button` | ✅       |
+| `onClick`  | `function` | `undefined` | Callback                               | ✅       |
+
+### Alert.SecondaryAction
+
+`Alert.SecondaryAction` renders and accepts a subset of `Link` (button) props:
+
+| Prop       | Type          | Default     | Description | Required |
+| ---------- | ------------- | ----------- | ----------- | -------- |
+| `children` | `string`, JSX | `undefined` | Content     | ✅       |
+| `href`     | `string`      | `undefined` | Hyperlink   |          |
+| `onClick`  | `function`    | `undefined` | Callback    |          |
+
+There'll also be an internal `BaseAlert` component that handles rendering web or native version similar to other Blade components. API would be similar to `Alert`.
+
+## Usage
+
+Some example usage patterns of `Alert`.
+
+### Without title
+
+<img src="./alert-without-title.png" alt="" width="520">
+
+```jsx
+<Alert variant="notice">
+  <Alert.Description>
+    The payment was made 6 months ago, therefore you can’t issue refund to this merchant.
+  </Alert.Description>
+</Alert>
+```
+
+### Full bleed
+
+> **Note**
+>
+> Design update pending to remove all borders and border radii
+
+This full bleed layout works for all desktop, mobile and native when the `isBorderless` prop is passed
```

Comment:
For setting width 100% we have the `isFullWidth` prop, so I think we should keep that decoupled from `isBorderless`.

We're not changing position CSS with `isFullWidth` 🤔 
I think absolute, relative shouldn't be needed because this will take entire width of container right? So users should customize the container's styles. Lemme know if I'm missing something.

Reply:
In general, what is the use case for Full Bleed ?

Reply:
For building end to end alerts - generally the ones which need to stick to the top of container / app. Example:

![image](https://user-images.githubusercontent.com/6682655/182312890-9767b6ea-4a1f-44f9-ad30-9cc0628d1943.png)

Reply:
Fullbleed (`isBorderless`) can be there without being full width (flat alerts). So kept both decoupled. Thoughts?

![image](https://user-images.githubusercontent.com/6682655/183021511-2853e983-8ff4-41f1-8c8f-02cff822bcf3.png)

![image](https://user-images.githubusercontent.com/6682655/183021661-db010267-0b2b-49f7-81c5-38f1401599fb.png)

Reply:
confirm with @saurav12 once. I'm okay whatever design takes the decision here

Reply:
@divyanshu013 I agree with @kamleshchandnani here. Full-bleed will be used with `fullWidth` only.

Reply:
Based on feedback, on design side whenever we specify `isBorderless`, it would automatically make the alert `isFullWidth`. Added an open question **Q6** for further info.